azure-pipelines.yml

```diff
@@ -334,7 +334,6 @@ jobs:
     - lint
     # The bootrom is built into the FPGA image at synthesis time.
     - sw_build_nexysvideo
-  condition: and(succeeded(), eq(dependencies.lint.outputs['DetermineBuildType.onlyDocChanges'], '0'), eq(dependencies.lint.outputs['DetermineBuildType.onlyDvChanges'], '0'))
```

Comment:
Can you leave that line in there, but only remove the `, eq(dependencies.lint.outputs['DetermineBuildType.onlyDvChanges']` part (the other part of the line is still needed). The same below.

Reply:
sure, done.